src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -283,13 +298,71 @@ export class Table extends PureComponent<TableProps, TableState> {
         'data-test-subj': 'updated-at',
         render: (updatedAt: string) => updatedAt && moment(updatedAt).format(dateFormat),
       } as EuiTableFieldDataColumnType<SavedObjectWithMetadata<any>>,
-      ...columnRegistry.getAll().map((column) => {
```

Comment:
We are not supposed to delete the columnRegistry here, it is an extension point for other plugins to register their customized column.

Reply:
Thanks for the comment, already updated